### DISCUSSION TITLE
Power Fx Microsoft Build 2023 announcements

### DISCUSSION THREAD

**Discussion by MikeStall** [microsoft/Power-Fx#1529] [General]
We had several major announcements at Build 2023 about Power Fx. 

These are all using the open-souce Power Fx implementation from this repo! 

## Power Virtual Agents is now GA!
Announcement: [Reinvent your AI assistants with generative answers, actions, and more in Microsoft Power Virtual Agents](https://powervirtualagents.microsoft.com/en-us/blog/reinvent-your-ai-assistants-with-generative-answers-actions-and-more-in-microsoft-power-virtual-agents/)

This also includes CLU integration: [Create bots with Azure cognitive language understanding (preview)](https://learn.microsoft.com/en-us/power-virtual-agents/advanced-clu-integration)

## Power Fx Plugins in Dataverse
Announcement: [Safely hyperscale low-code apps with ease using Microsoft Dataverse](https://powerapps.microsoft.com/en-us/blog/safely-hyperscale-low-code-apps-with-ease-using-microsoft-dataverse/)

Docs: [Example Dataverse low-code plug-ins (experimental)](https://learn.microsoft.com/en-us/power-apps/maker/data-platform/lowcode-plug-ins-examples)

## Power Cards Updates
Announcement: https://powerapps.microsoft.com/en-us/blog/easily-design-and-automatically-send-actionable-data-driven-cards-in-teams/

## Power Fx 1.0 GA for all apps
We are very close to publishing the nugets for Power Fx 1.0. Any app will be able to easily build low-code features via Power Fx in their own app. 
See Joris's session at [Accelerate development with Visual Studio and Microsoft Power Platform](https://build.microsoft.com/en-US/sessions/a2154a8a-57fe-4717-af9a-bea7ec2b5e94?source=sessions)  (start at 15 min)

## Updated Formula bar in Power Apps
Power Apps is adopting the same Power Fx Formula Bar published on NPM used in all other products (often called our "universal formula bar"): [Try the new Power Fx Formula Bar in Power Apps Studio](https://powerapps.microsoft.com/en-us/blog/try-the-new-power-fx-formula-bar-in-power-apps-studio/)

**Comment by jas-valgotar** [comment]
[celebrate]     Jas Valgotar reacted to your message:
________________________________
From: Mike Stall ***@***.***>
Sent: Thursday, May 25, 2023 6:08:14 PM
To: microsoft/Power-Fx ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [microsoft/Power-Fx] Power Fx Microsoft Build 2023 announcements (Discussion #1529)


We had several major announcements at Build 2023 about Power Fx.

These are all using the open-souce Power Fx implementation from this repo!

Power Virtual Agents is now GA!

Announcement: Reinvent your AI assistants with generative answers, actions, and more in Microsoft Power Virtual Agents<https://powervirtualagents.microsoft.com/en-us/blog/reinvent-your-ai-assistants-with-generative-answers-actions-and-more-in-microsoft-power-virtual-agents/>

This also includes CLU integration: Create bots with Azure cognitive language understanding (preview)<https://learn.microsoft.com/en-us/power-virtual-agents/advanced-clu-integration>

Power Fx Plugins in Dataverse

Announcement: Safely hyperscale low-code apps with ease using Microsoft Dataverse<https://powerapps.microsoft.com/en-us/blog/safely-hyperscale-low-code-apps-with-ease-using-microsoft-dataverse/>

Docs: Example Dataverse low-code plug-ins (experimental)<https://learn.microsoft.com/en-us/power-apps/maker/data-platform/lowcode-plug-ins-examples>

Power Cards Updates

Announcement: https://powerapps.microsoft.com/en-us/blog/easily-design-and-automatically-send-actionable-data-driven-cards-in-teams/

Power Fx 1.0 GA for all apps

We are very close to publishing the nugets for Power Fx 1.0. Any app will be able to easily build low-code features via Power Fx in their own app.

**Comment by rajyraman** [comment]
@MikeStall - Just watched the part with Joris. Never knew about the marshalling bit. Amazing stuff!👏
